stl/inc/yvals.h

```diff
@@ -230,12 +230,10 @@ _STL_DISABLE_CLANG_WARNINGS
 
 #include <use_ansi.h>
 
-#define _WARNING_MESSAGE(NUMBER, MESSAGE) __FILE__ "(" _CRT_STRINGIZE(__LINE__) "): warning " NUMBER ": " MESSAGE

```

Comment:
Note the stealth merge conflict with #2061. (Both PRs are deleting these lines, but #2061 moves them to `yvals_core.h` and introduces many uses.)